projects/demo/src/modules/components/data-list/examples/2/index.html

```diff
@@ -1,4 +1,4 @@
-<tui-hosted-dropdown #dropdown [content]="content">
+<tui-hosted-dropdown #dropdown [content]="content" [(open)]="dropdownOpened">
```

Comment:
Let's use `open`. Because `openED` is a past form of actual verb `open`, not a state of the thing. That's why we switched from `opened` to `open` in 2.0 :)

Reply:
fixed